model/fieldtypes/compat/Float.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
```

Comment:
See my comment above. My major dislike is the roles that the php-7 equivalent fields in trying to load these deprecated classes.